include/deal.II/sundials/kinsol.h

```diff
@@ -664,6 +681,14 @@ namespace SUNDIALS
      */
     void *kinsol_mem;
 
+#  if DEAL_II_SUNDIALS_VERSION_GTE(5, 0, 0)
+    /**
+     * An object that describes the linear solver, in case the user decides
+     * to attach one. Otherwise it remains unused.
+     */
+    SUNLinearSolver sun_linear_solver;
```

Comment:
Seems to be unused.

Reply:
Good eyes -- that was a left-over from one of my previous iterations...